app/templates/vendor/orders.html

```diff
@@ -105,7 +108,22 @@ <h2 class="ui header">
         {% endif %}
 
     </div>
+    <div class="ui modal">
+        <div class="header">You are about to <b class="action-type"></b> this order:</div>
+        <div class="content">
+            <div class="ui form">
+                  <div class="field">
+                    <label>Please write any comments to be sent to the customer <em>(optional)</em>:</label>
+                    <textarea rows="2" class="comment-container"></textarea>
+                  </div>
+            </div>
+        </div>
 
+        <div class="confirm-action">
+            <button class="negative ui button tiny compact cancel">Cancel</button>
```

Comment:
I mean the ordering of the 'Cancel' and 'Approve' buttons in your second screenshot from the Vendor side. But now I see that on your branch I was testing, the buttons were aligned to the left which made it seem weird to me that the 'Cancel' button was before the 'Approve' but now that you aligned it to the right, it's fine.